engine/runtime/src/main/java/org/enso/interpreter/runtime/data/Type.java

```diff
@@ -208,16 +222,24 @@ String toDisplayString(boolean allowSideEffects) {
   }
 
   @ExportMessage
-  boolean isMetaObject() {
+  boolean isMetaObject(@CachedLibrary("this") InteropLibrary lib) {
+    var b = EnsoContext.get(lib).getBuiltins();
+    if (this == b.nothing()) {
+      return false;
+    }
     return true;
```

Comment:
If we really want to have this then please refactor this for DRY

Reply:
Useless, but fine: 11710c4462